{"name": "test_login_params[data_for_login3]", "status": "passed", "attachments": [{"name": "log", "source": "0098c696-192a-4117-be66-e074efd9f346-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data_for_login", "value": "('adelukesh24@gmail.com', '54851', 'Fail')"}], "start": 1747036250995, "stop": 1747036254007, "uuid": "966de63b-5ef6-41e7-bcbd-78d797916004", "historyId": "d86d1153d36b45d62101e56b53503bb5", "testCaseId": "47f970e602e4eefb1a3e5455dd555a36", "fullName": "testCases.test_login_params.TestLoginParams#test_login_params", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup', 'log_on_failure')"}, {"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_login_params"}, {"name": "subSuite", "value": "TestLoginParams"}, {"name": "host", "value": "lukesh"}, {"name": "thread", "value": "27268-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_login_params"}]}